openapi/components/schemas/DepositRequest.yaml

```
@@ -125,6 +125,21 @@ properties:
     example:
       email: email@example.com
       max: '33'
+  notificationUrl:
+    description: |-
+      URL where a server-to-server POST notification is sent.
+      This notification is sent when the transaction result is finalized after a timeout or an offsite interaction.
+
+      Do not trust this notification alone,
+      complete a GET request to confirm the result of the transaction.
+      To ensure the request is not reattempted,
+      when the result is confirmed, respond with a 2xx HTTP status code.
+
+      The following placeholders are available to use in this URI: `{id}` and `{result}`.
```

suggestion: we should use the same text if possible

'Do not trust this notification alone' is an ambiguous statement.

```suggestion
      Do not interpret this notification as a confirmation,
      complete a `GET` request to confirm the result of the transaction.
      To ensure the request is not reattempted,
      when the result is confirmed, respond with a `2xx` HTTP status code.

      The following placeholders are available to use in this URI: `{id}` and `{result}`.
      These placeholders are replaced the with the transaction ID and result accordingly.
```

---

Hi @noon-dawg this description was taken from Tranasaction.notification

---

Hi @ashkarpetin, thanks for spotting this 🙏 
I've updated the the description in Tranasaction.notication in this [PR](https://github.com/Rebilly/api-definitions/pull/1764)

---

I am going to merge this, can you please update this description in that PR too